<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Settings</class>
 <widget class="QDialog" name="Settings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>347</width>
    <height>266</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>330</width>
    <height>200</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <widget class="QWidget" name="playerTab">
      <attribute name="title">
       <string>Player</string>
      </attribute>
      <layout class="QFormLayout" name="formLayout">
       <property name="fieldGrowthPolicy">
        <enum>QFormLayout::ExpandingFieldsGrow</enum>
       </property>
       <item row="5" column="0" colspan="2">
        <widget class="QCheckBox" name="resizeCheckBox">
         <property name="text">
          <string>Resize window automatically</string>
         </property>
        </widget>
       </item>
       <item row="6" column="0" colspan="2">
        <widget class="QCheckBox" name="screenshotCheckBox">
         <property name="toolTip">
          <string>Need screenshot filter (-vf-add screenshot)</string>
         </property>
         <property name="text">
          <string>Enable screenshot</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QCheckBox" name="skinCheckBox">
         <property name="text">
          <string>Skin</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QComboBox" name="skinComboBox"/>
       </item>
       <item row="7" column="0" colspan="2">
        <widget class="QCheckBox" name="rememberCheckBox">
         <property name="text">
          <string>Remember unfinished clips</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="videoTab">
      <attribute name="title">
       <string>Video</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <layout class="QHBoxLayout" name="voLayout">
         <item>
          <widget class="QLabel" name="voLabel">
           <property name="text">
            <string>Video Output</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="voComboBox">
           <item>
            <property name="text">
             <string>xv</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>vdpau</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>vaapi</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>directx</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>direct3d</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>gl</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>x11</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QStackedWidget" name="voStackedWidget">
         <widget class="QWidget" name="page">
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QLabel" name="xvLabel">
             <property name="text">
              <string>The default output in Linux. Cannot be used in Windows.</string>
             </property>
             <property name="wordWrap">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="page_7">
          <layout class="QVBoxLayout" name="verticalLayout_7">
           <item>
            <widget class="QLabel" name="vdpauLabel">
             <property name="text">
              <string>Nvidia hardware decoding in Linux.</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="ffodivxvdpauCheckBox">
             <property name="text">
              <string>Enable ffodivxvdpau</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="page_2">
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="QLabel" name="vaapiLabel">
             <property name="text">
              <string>Enable Intel/Ati hardware decoding in Linux. Cannot be used in Windows.</string>
             </property>
             <property name="wordWrap">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="page_3">
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <widget class="QLabel" name="dxLabel">
             <property name="text">
              <string>Default output in Windows. Cannot be used in Linux.</string>
             </property>
             <property name="wordWrap">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="page_4">
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <item>
            <widget class="QLabel" name="d3Label">
             <property name="text">
              <string>Default output in Windows Vista and Windows 7. Cannot be used in Linux and Windows XP.</string>
             </property>
             <property name="wordWrap">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="page_5">
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QLabel" name="glLabel">
             <property name="text">
              <string>OpenGL Output in Linux. You can try it if xv does not work.</string>
             </property>
             <property name="wordWrap">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="page_6">
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QLabel" name="x11Label">
             <property name="text">
              <string>Output in Linux. You can try it if xv does not work.</string>
             </property>
             <property name="wordWrap">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="doubleCheckBox">
         <property name="text">
          <string>Double buffer</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="dropCheckBox">
         <property name="text">
          <string>Allow framedrop</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="fixCheckBox">
         <property name="text">
          <string>Try fixing &quot;stop at last frame&quot; problem</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="netTab">
      <attribute name="title">
       <string>Internet</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_8">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="proxyLabel">
           <property name="text">
            <string>Http proxy</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="proxyEdit"/>
         </item>
         <item>
          <widget class="QLabel" name="portLabel">
           <property name="text">
            <string>Port</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="portEdit">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="maximumSize">
            <size>
             <width>50</width>
             <height>16777215</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Video quality:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="normalRadioButton">
           <property name="text">
            <string>Normal</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="highRadioButton">
           <property name="text">
            <string>High</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="superRadioButton">
           <property name="text">
            <string>Super</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="Line" name="line_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QFormLayout" name="formLayout_2">
         <property name="fieldGrowthPolicy">
          <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
         </property>
         <item row="0" column="0">
          <widget class="QLabel" name="maxTaskLabel">
           <property name="text">
            <string>Max tasks</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QSpinBox" name="maxTaskSpinBox">
           <property name="minimum">
            <number>1</number>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Save to:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QPushButton" name="dirButton"/>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Cache(KB):</string>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Cache min:</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QSpinBox" name="cacheMinSpinBox">
           <property name="maximum">
            <number>100</number>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QSpinBox" name="cacheSpinBox">
           <property name="maximum">
            <number>20480</number>
           </property>
          </widget>
         </item>
         <item row="2" column="0" colspan="2">
          <widget class="Line" name="line">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Settings</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Settings</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>voComboBox</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>voStackedWidget</receiver>
   <slot>setCurrentIndex(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>184</x>
     <y>97</y>
    </hint>
    <hint type="destinationlabel">
     <x>123</x>
     <y>112</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
